packages/widget-v2/tsconfig.app.json

```diff
@@ -26,7 +26,7 @@
     /* Aliases */
     "baseUrl": ".",
     "paths": {
-      "@*": ["src/*"]
+      "@/*": ["src/*"]
```

Comment:
This fixes auto imports / ide from suggesting `@components` instead of `@/components` which isn't valid with this config